Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.cs

```diff
@@ -381,6 +373,27 @@ public void AddTokenItem(object data, bool atEnd = false)
             _ = AddTokenAsync(data, atEnd);
         }
 
+        /// <summary>
+        /// Clears the whole collection, will raise the <see cref="TokenItemRemoving"/> event asynchronously for each item.
+        /// </summary>
+        /// <returns>async task</returns>
+        public async Task Clear()
```

Comment:
I would make this ClearAsync, since it returns a Task.

Reply:
fixed